ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/graphics/ros2/RDXROS2FramePlanarRegionsVisualizer.java

```diff
@@ -115,4 +115,10 @@ public void destroy()
 
       planarRegionsGraphic.destroy();
    }
+
+   public void setOpacity(double opacity)
+   {
+      this.opacity.set(opacity);
+      planarRegionsGraphic.setBlendOpacity((float) opacity);
+   }
```

Comment:
This is used so we can show the planar regions by default in the scene graph demo without them getting in the way.